modules/editor/assets/stylesheets/main.scss

```diff
@@ -1,12 +1,19 @@
+@import '../../inserter/style';
+
 .gutenberg {
 	background: #fff;
 	margin: -20px 0 0 -20px;
 	padding: 60px;
+
+	* {
+		box-sizing: border-box;
```

Comment:
In our prototype @intronic and me have been using CSS modules which allows for a better style encapsulation and maintainability. I think it's a step above BEM and it would be worth trying. A first step towards this direction would be to start migrating some sections such this one.